src/modules/UI/scenes/SendConfirmation/action.js

```diff
@@ -50,8 +110,7 @@ export const updateMaxSpend = () => (dispatch: Dispatch, getState: GetState) =>
 
   getMaxSpendable(edgeWallet, spendInfo)
     .then(nativeAmount => {
```

Comment:
I think that this is prettier but you don't have to change it:
then(nativeAmount => dispatch(createTX({ nativeAmount }, true)))

Reply:
- update https://github.com/Airbitz/edge-conventions with your preference
- find a corresponding eslint rule
- update `.eslintrc`

Reply:
agreed, but there are some types that expected `void` as the return value, i dont know if thats the case here though.

Reply:
https://eslint.org/docs/rules/arrow-body-style
'arrow-body-style': ['error', 'as-needed'],